### ISSUE TITLE
Strange, rare error when setting Evidence PMID

### ISSUE THREAD

**Issue by travs** [openworm/PyOpenWorm#210]
I've only encountered this once, but when I do this:
```python
# example.py
import PyOpenWorm as P
P.connect()
ev = P.Evidence(pmid='21177967')
print(ev) # or save(), or any other operation with "ev"
```
I get this error:
```
Traceback (most recent call last):
  File "example.py", line 4, in <module>
    print(ev)
  File "/Users/travis/.virtualenvs/channelworm/lib/python2.7/site-packages/PyOpenWorm-0.5.3-py2.7.egg/PyOpenWorm/dataObject.py", line 99, in __str__
    s +=  ", ".join(str(x) for x in self.properties if x.hasValue())
  File "/Users/travis/.virtualenvs/channelworm/lib/python2.7/site-packages/PyOpenWorm-0.5.3-py2.7.egg/PyOpenWorm/dataObject.py", line 99, in <genexpr>
    s +=  ", ".join(str(x) for x in self.properties if x.hasValue())
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe4' in position 32: ordinal not in range(128)
```
**What?**

From my experience, this *only* happens with this particular PMID, and not with adjacent PMIDs '21177966' or '21177968'.
Also note that using a unicode, raw or regular string as the input doesn't affect this behaviour.

Anyway, the workaround I'm using is to not set the PMID when initializing the Evidence object, but to set it afterward. Like so:
```python
ev = P.Evidence()
ev.pmid('21177967')
print(ev) # do whatever with 'ev' now
```

Weird stuff... :confused:

**Comment by slarson** [comment]
Looks like line 99 of dataObject.py needs somewhere to have `.encode('UTF-8')` thrown in: https://docs.python.org/2/howto/unicode.html#the-unicode-type

**Comment by mwatts15** [comment]
@slarson Good suggestion.
@signija Please, have a look at my pull request #250.

**Comment by signija** [comment]
@mwatts15 Messaged you about it. As far as I can tell, the solution would be doing `u"Frédéric MY".encode('UTF-8')` for the first argument instead of only `u"Frédéric MY"`, and then the build should pass. I just don't know exactly why, I was trying out things to figure out what was happening, and I found out that's what it's doing.

**Comment by mwatts15** [comment]
Fixed by PR #279.
